crates/connectors/ndc-postgres/src/explain.rs

```diff
@@ -89,23 +89,27 @@ fn plan_query(
 ) -> Result<sql::execution_plan::ExecutionPlan<sql::execution_plan::Query>, connector::ExplainError>
 {
     let timer = state.metrics.time_query_plan();
-    let result =
-        translation::query::translate(&configuration.metadata, query_request).map_err(|err| {
-            tracing::error!("{}", err);
-            match err {
-                translation::error::Error::CapabilityNotSupported(_) => {
-                    state.metrics.error_metrics.record_unsupported_capability();
-                    connector::ExplainError::UnsupportedOperation(err.to_string())
-                }
-                translation::error::Error::NotImplementedYet(_) => {
-                    state.metrics.error_metrics.record_unsupported_feature();
-                    connector::ExplainError::UnsupportedOperation(err.to_string())
-                }
-                _ => {
-                    state.metrics.error_metrics.record_invalid_request();
-                    connector::ExplainError::InvalidRequest(err.to_string())
-                }
+    let result = translation::query::translate(
+        &configuration.metadata,
+        &sql::ast::transaction::IsolationLevel::default(),
```

Comment:
A somewhat academic question perhaps: Can the transaction isolation level affect the query plan?

As a hot take I would think it wouldn't, but as a matter of being conservative it might be good to use the isolation level specified in the configuration still.

Reply:
Fair point. I will add a new ticket for this.

Reply:
Oh I will change this now, but we should probably run `EXPLAIN` in a transaction as well to make sure the postgres query plan is the same.